README.md

```diff
@@ -1,14 +1,38 @@
 # freegeek-info-script
-Shell script for getting info on Linux Mint needed for Freegeek build sheets
+Shell script for Linux Mint that conveniently pools together info needed for Free Geek build sheets.
 
-How to run:
+### How to run:
+1. Type `Ctrl + Alt + T` to open a terminal
```

Comment:
```suggestion
1. Open a terminal
```
The terminal is on the dock by default in Linux Mint

Reply:
Can say open a terminal with the icon on the dock, but saying to open a terminal a certain way (that most people don't use) seems a bit off.

If they can't figure out what the terminal is they shouldn't be doing this.